about.html

```diff
@@ -0,0 +1,169 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="about.css" />
+    <link href="https://fonts.googleapis.com/css?family=lato" rel="stylesheet"/>
+    <title>Capstone Project 1</title>
+  </head>
+  <body>
+    <div class="mobile-menu" id="mobile-menu">
+      <div class="mobile-menu-header">
+        <button type="button" class="close-button">&times;</button>
+      </div>
+      <nav>
+        <a href="./index.html" class="navigate"> Home </a>
+        <a href="./about.html" class="navigate"> About </a>
+        <a href="#contact" class="navigate"> Contact </a>
+        <a href="#previous-summits" class="navigate">Summits </a>
+        <a href="./about.html#sponsor" class="navigate"> Sponsor </a>
+      </nav>
+    </div>
```

Comment:
You did well in implementing the mobile menu, But I should be able to navigate to the about page and back to the home page using the mobile menu. please fix this.